src/main/webapp/WEB-INF/views/index.jsp

```diff
@@ -0,0 +1,22 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<style>
+    .rotatable{
+        -webkit-transform: rotate(-45deg);
+        -moz-transform: rotate(-45deg);
+        -o-transform: rotate(-45deg);
+        -ms-transform: rotate(-45deg);
+        transform: rotate(-45deg);
+        margin-top: 4em;
+    }
+</style>
+<head>
+    <title>Title</title>
+</head>
+<body>
+<div class='rotatable'>
+    <h1 style="text-align: center; color: red;">Hello world!!!</h1>
+</div>
+
```

Comment:
Maybe you should remove the extra newline here

Reply:
done